app/src/main/java/com/github/kiolk/cowsandbulls/utils/ThemeHelper.java

```diff
@@ -7,11 +7,11 @@
 
 public class ThemeHelper {
```

Comment:
In switch you can use standard constants
AppCompatDelegate.MODE_NIGHT_FOLLOW_SYSTEM 
AppCompatDelegate.MODE_NIGHT_NO
AppCompatDelegate.MODE_NIGHT_YES 

Same, this values you need save in prefs